paths_test.go

```diff
@@ -12,12 +12,17 @@ import (
 
 func TestPaths(t *testing.T) {
 	const k = "APPDATA"
+	const VersionAstilectron = DefaultVersionAstilectron
+	const VersionElectron = DefaultVersionElectron
+
 	ad := os.Getenv(k)
 	os.Setenv(k, "")
 	ep, err := os.Executable()
 	ep = filepath.Dir(ep)
 	assert.NoError(t, err)
-	p, err := newPaths("linux", "amd64", Options{})
+
+	o := Options{VersionAstilectron: DefaultVersionAstilectron, VersionElectron: DefaultVersionElectron}
```

Comment:
Can you not create this `o` variable and put the `Options` struct directly in the `newPath` call below?

Reply:
Left since Options{} being passed to newPaths() would cause issue since versions would not be set. reused Options o passing to multiple tests.